src/plone/restapi/search/handler.py

```diff
@@ -75,6 +75,10 @@ def _constrain_query_by_path(self, query):
             path = "/".join(self.context.getPhysicalPath())
             query["path"]["query"] = path
 
+    def quote_chars(self, query):
+        # Escape parentheses by adding backslashes before them
```

Comment:
This is not escaping, but removing.

```suggestion
        # Remove parentheses from the query
```